pi_server/pi_server/__main__.py

```diff
@@ -0,0 +1,4 @@
+from pi_server import app
+
+if __name__ == "__main__":
+    app.run(host="0.0.0.0", port=8080)
```

Comment:
Please use a clean conf management:
https://flask.palletsprojects.com/en/1.1.x/config/
https://stackoverflow.com/questions/20212894/how-do-i-get-flask-to-run-on-port-80